applylib/applyset/results.go

```diff
@@ -27,6 +27,8 @@ type ApplyResults struct {
 	total             int
 	applySuccessCount int
 	applyFailCount    int
+	pruneSuccessCount int
+	pruneFailCount    int
```

Comment:
We probably need to include pruneFailCount in AllApplied ... AllApplied is sort of considered a "did we apply everything OK", and if we are now pruning as part of apply, we want that to be false if prune failed, so that we can surface the information that we aren't completely done yet.  (?)

Reply:
Yes, I think we should treat pruning failures like other failures.  We might want to surface the status specially, but we shouldn't pretend everything is OK.